### PR TITLE
Dart SDK stack and initial Flutter SDK support

### LINKED ISSUE #12385: Dart and Flutter Stacks
### Description

Goes hand in hand with #12370.

Default stacks for Dart would be:

- the Dart SDK only

- Dart SDK with Flutter with:
   - Android SDK

I will be writing the defualt stacks for such since they're gonna be needed for the referenced PR as well.

### PR DESCRIPTION
**Pull request by sr229**
Signed-off-by: Capuccino <enra@sayonika.moe>

### What does this PR do?

Adds the Flutter (No VNC) and Dart Stack

### What issues does this PR fix or reference?

Implemented as part of eclipse/che#12370

Fixes eclipse/che#12385

### New behavior

Adds Dart and Flutter stacks as initial support.

### Tests written?
No. 

### Docs updated?
No.

### CODE REVIEW

**Comment by dneary** [comment]
@eivantsov Can you review this please?

File: recipes/dart/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2012-2018 Red Hat, Inc.
```
Comment (monaka) [inline recipes/dart/Dockerfile]: @sr229 Let me make sure. Are you working in Red Hat?

Comment (sr229) [reply recipes/dart/Dockerfile]: Oh this was a copy of the header from the rest of the images (I based it off the Android stack). I can remove it if such.

Comment (monaka) [reply recipes/dart/Dockerfile]: @sr229 I read `recipes/android/Dockerfile` and I think your code is almost "from scratch".
@eivantsov How do you think?

**Review by ghost — APPROVED** [review]

**Pull request closed without merging**